packages/devextreme/js/__internal/ui/scroll_view/utils/get_translate_values.test.tsx

```diff
@@ -1,8 +1,9 @@
-import { getTranslateValues } from '../get_translate_values';
-import { getElementTransform } from '../get_element_style';
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
Do we really need that comment here?

Reply:
Unfortunately, we do
`jest.requireActual` in code below returns any and this lint rule fails

<img width="927" alt="image" src="https://github.com/user-attachments/assets/c2e4230f-24b4-4f76-8092-b55b47170388">
<img width="519" alt="image" src="https://github.com/user-attachments/assets/f81b6a27-e0d9-4154-b1bf-05bd4f8f1c66">